executor/mock/jobrunner.go

```diff
@@ -252,7 +282,9 @@ func (jobRunner *JobRunner) StartJob(jobInput *JobInput) *JobRunResponse { // no
 	}
 
 	// TODO: refactor this all to use NewContainer()
-	taskID := fmt.Sprintf("Titus-%v%v-Worker-0-2", r.Intn(1000), time.Now().Second())
+	validContainerNameRE := regexp.MustCompile("[^a-zA-Z0-9_.-]")
```

Comment:
Wait, why do we do this replacement?

Reply:
This strips out characters that docker won't allow in container names.  Will add a comment.